galata/test/documentation/utils.ts

```diff
@@ -38,7 +38,7 @@ export function generateArrow(
  * @returns The svg to inject in the page
  */
 export function positionMouse(position: { x: number; y: number }): string {
```

Comment:
I was thinking if we should make the signature something like `position: { x: number | ElementHandle, y: number | ElementHandle }` to allow setting one of the coordinates manually while the other would be in the middle of BBox of the element. We can extract this to a new issue (along with changing the cursor to something nicer, and adding it to more screenshots).

Reply:
Nice suggestion.